docs/docs/channels/sms/telnyx.md

```diff
@@ -9,7 +9,7 @@ To use the Telnyx channel, you will need to create a Telnyx account and add your
 ## Find the API Key
 
 To find your Telnyx API key: 
-- Log in to your Telnyx account.
+- Log into your Telnyx account.
```

Comment:
in general, corrections for `into` have been proven to be quite controversial. I'm happy to drop them (or anything else).